### DISCUSSION TITLE
Are firefox and chrome version built differently ?

### DISCUSSION THREAD

**Discussion by GGLinnk** [openstyles/stylus#1659] [General]
Hello,

I've installed a stylus theme for stylus.
It have been designed for chrome and it's working fine on it.
But changing the prefix doesn't seems to be enough to make it work.

![image](https://github.com/openstyles/stylus/assets/22531645/a277f7dc-5b57-4a81-a24f-fb6440e3f825)
![image](https://github.com/openstyles/stylus/assets/22531645/43fd31ed-a6ef-4bc4-9914-4ec84fe27f15)

So my question are :
Is that normal behavior ?
Is it a difference in how the extension is built or is it because of the browser ?

**Comment by tophf** [comment]
It's hard to say without seeing the userstyle, but assuming the version of Stylus is the same the JS code is identical as well as 99.9% of CSS, the remaining 0.1% may be different to compensate for the bugs in the browsers, which you should see by inspecting the elements in devtools.
